src/modules/UI/scenes/Request/reducer.js

```diff
@@ -57,19 +55,7 @@ export const request = (state: RequestSceneState = initialState, action: Action)
       }
     }
 
-    case ACTION.UPDATE_METADATA: {
-      const { receiveAddress } = state
-      const { metadata = {} } = receiveAddress
-      return {
-        ...state,
-        receiveAddress: {
-          ...receiveAddress,
-          metadata
-        }
-      }
-    }
-
-    case ACTION.UPDATE_AMOUNT_REQUESTED_IN_FIAT: {
+    case 'UPDATE_AMOUNT_REQUESTED_IN_FIAT': {
```

Comment:
We could probably PR this separately so we can verify that this change is correct.